content/rs/references/rladmin.md

```diff
@@ -1,30 +1,697 @@
+| persistent_path | Path to persistent storage location |
+| register_nds_suffix | Enables database mapping to both internal and external IP |
+| flash_enabled | Enables flash storage |
+| flash_path | Path to flash storage location |
+| addr | Internal IP addresses of the node |
+| external_addr | External IP addresses of the node |
+
+#### `cluster join`
+
+`rladmidn cluster join` adds a node to an existing cluster. 
+
+```text
+rladmin cluster join 
+        name <cluster-name> | nodes <node-address> 
+        username <admin-user> 
+        password <admin-password>
+        [ ephemeral_path <path> ]
+        [ persistent_path <path> ]
+        [ rack_id <node-rack-id> ]
+        [ override_rack_id ]
+        [ replace_node <node-uid> ]
+        [ flash_enabled ]
+        [ flash_path <path> ]
+        [ addr <ip-address> ]
+        [ external_addr <ip-addresses> ]
+        [ override_repair ]
+        [ cmn_http_port <port> ]
+```
+
+| Optional Parameters | Description |
+| - | - |
+| ephemeral_path | Path to ephemeral storage location |
+| persistent_path | Path to persistent storage location |
+| rack_id | Rack ID of the rack |
+| override_rack_id | Manually overrides the existing default rack ID |
+| replace_node | Replaces specified node with new node |
+| flash_enabled | Enables flash storage |
+| flash_path | Path to flash storage location |
+| addr | Internal IP addresses of the node |
+| external_addr | External IP addresses of the node |
+| override_repair | Enables joining a cluster with a dead node |
+| cmn_http_port | Joins a cluster that has a non-default cnm_http_port |
+
+#### `cluster recover`
+
+`rladmin cluster recover` recovers a cluster from a backup file.
+
+```text
+rladmin cluster recover 
+        filename <recovery-file-name> 
+        [ ephemeral_path <path> ] 
+        [ persistent_path <path> ] 
+        [ rack_id <ID> ] 
+        [ override_rack_id ] 
+        [ node_uid <number> ] 
+        [ flash_enabled ] 
+        [ flash_path <path> ] 
+        [ addr <ip-address> ] 
+        [ external_addr <ip-addresses> ]
+```
+
+| Optional Parameters | Description |
+| - | - |
+| ephemeral_path | Path to ephemeral storage location |
+| persistent_path | Path to persistent storage location |
+| rack_id | Rack ID of the rack |
+| override_rack_id | Manually overrides the existing default rack ID |
+| flash_enabled | Enables flash storage |
+| flash_path | Path to flash storage location |
+| addr | Internal IP addresses of the node |
+| external_addr | External IP addresses of the node |
+
+### `failover`
+
+`rladmin failover` performs a failover for master shards of a specific database, and promotes their respective replicas to new masters. The `immediate` flag performs the failover without verifying the replica shards are in full sync with the master shards.
+
+```text
+rladmin failover 
+        [ db <db:id | name> ] 
+        shard <id1 .. idN> 
+        [ immediate ]
+```
+
+### `help`
```

Comment:
@kaitlynmichaelredislabs If I may suggest - can we start with the Help and Info commands and then move up in order of complexity/impact/danger?

Reply:
That's an interesting question. If they are organized by difficulty and not alphabetically, this page might be more difficult to navigate. Do we know if there is an industry standard for how to organize command reference material?

Reply:
Here are a couple of references I'm familiar with and I think are pretty well organized.
https://docs.microsoft.com/en-us/cli/azure/get-started-with-azure-cli
https://docs.openshift.com/container-platform/4.7/cli_reference/openshift_cli/getting-started-cli.html

I really like how the Azure guide explains what it is, getting started, and then points to popular uses. Maybe we need to separate these from the alphabetized reference guide.